### DISCUSSION TITLE
Where is Faceswap

### DISCUSSION THREAD

**Discussion by Wontfallo** [lllyasviel/Fooocus#2196] [Q&A] [answered]
I don't see faceswap.  If this has been addressed or answered please forgive me and fix your front page, still shows
If it's in a new location can you please point me to it, and also fix your front page.  
If it's still there and I'm looking right at it, please just tell me I'm a silly dumb dumb.  
I don't see it here:   Input Image -> Image Prompt -> Advanced -> FaceSwap

Also I know you can run plenty of faceswap options in automatic1111111111111111111111111111 and I do, but Im just going off the docs here and trying to use Fooocus instead, 

Thanks!

```
<html>
<body>
<!--StartFragment-->
InsightFace | Input Image -> Image Prompt -> Advanced -> FaceSwap
-- | --


<!--EndFragment-->
</body>
</html>
```

**Comment by xhoxye** [comment]
![QQ截图20240207102837](https://github.com/lllyasviel/Fooocus/assets/129571231/6e6533ed-63f9-4545-83b9-06a2b9f44559)

**Reply by Wontfallo** [reply, marked as answer]
Mine doesn't show that. What am I doing different? I also have Sketch and you do not.

**Comment by Wontfallo** [comment]
![image](https://github.com/lllyasviel/Fooocus/assets/93687550/96a30faf-e731-40a0-8b63-119493428f6e)

**Reply by Wontfallo** [reply]
![SayWhatDunnoGIF](https://github.com/lllyasviel/Fooocus/assets/93687550/078c9f50-bf57-43b9-b928-ad4fac257d08)

**Comment by xhoxye** [comment]
You're looking in the wrong place. The version here is different from yours.

**Reply by Wontfallo** [reply]
Interesting. I will have to investigate why I had that other version and where I got it from. Was it a fork or just an older version? I thought I got it here. Weird. Anyways thanks for the speedy help friend. I do always appreciate it.

**Reply by mashb1t** [reply]
@Wontfallo you're using Fooocus control https://github.com/fenneishi/Fooocus-ControlNet-SDXL, which has not been updated since its creation over 4 months ago and is behind 230+ commits. If you want to use Fooocus core features please make sure to reinstall.

**Comment by Wontfallo** [comment]
Ah yes that makes sense. Thank you so much for your help. This is what i
was looking for originally. Thanks again great work to
